src/views/Correction.vue

```diff
@@ -262,21 +262,11 @@ export default {
     },
 
     /** Returns title of original filing. */
-    title (): string | null {
+    title (): string {
       if (this.origFiling && this.origFiling.header && this.origFiling.header.name) {
-        switch (this.origFiling.header.name) {
-          case FilingTypes.ANNUAL_REPORT: return FilingNames.ANNUAL_REPORT + (this.agmYear ? ` (${this.agmYear})` : '')
-          case FilingTypes.CHANGE_OF_ADDRESS: return FilingNames.ADDRESS_CHANGE
-          case FilingTypes.CHANGE_OF_DIRECTORS: return FilingNames.DIRECTOR_CHANGE
-          case FilingTypes.CHANGE_OF_NAME: return FilingNames.LEGAL_NAME_CHANGE
-          case FilingTypes.SPECIAL_RESOLUTION: return FilingNames.SPECIAL_RESOLUTION
-          case FilingTypes.VOLUNTARY_DISSOLUTION: return FilingNames.VOLUNTARY_DISSOLUTION
-          case FilingTypes.CORRECTION: return FilingNames.CORRECTION
-        }
-        // fallback for unknown filings
-        return this.origFiling.header.name.split(/(?=[A-Z])/).join(' ').replace(/^\w/, c => c.toUpperCase())
+        return this.typeToTitle(this.origFiling.header.name)
```

Comment:
Please pass in `this.agmYear` parameter.

Reply:
Good catch, thanks!